ReactAndroid/src/androidTest/js/SubviewsClippingTestModule.js

```diff
@@ -54,6 +54,7 @@ class ClippingSample1 extends React.Component {
   }
 }
 
+/* eslint-disable react-native/no-unused-styles */
```

Comment:
It seems like we need these because the examples use a local alias like `var styles = sample2Styles;`. I haven't looked too closely but does it make sense to you why they do that and why we should ignore the rule for these blocks? Or should we remove the aliases and have these blocks referenced directly without the need for the lint disable?

Reply:
Yes, I think removing aliases is a better option. I'll work on that.